### ISSUE TITLE
Instructions

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#574]
Issue group: Create a service
Visibility: 2
Evaluation: Does not support
Success Criteria: 3.3.2: Labels or Instructions
Needs design: Content

### Description
When setting up a new service, a sending email address is required. While visually it is clear that the username portion of the email address is to be inputted, this is not well communicated to a non-visual user. The “@notification.canada.ca” domain is not associated with the input field, other than through location and visual positioning, and the instructions only define the entry as an email address. Provide this information in the instructions and associate with the input field using aria-describedby.

![](https://dl.airtable.com/.attachments/7dc2b6cac07d1006d6f0714aa5678a33/fa49aea0/Capturedecranle2022-03-15a13_53_34.png)

**Comment by amazingphilippe** [comment]
This was the main focus of this story: #606
